config/test.yml.erb

```diff
@@ -65,3 +65,6 @@ statsig_api_client_key: 'client-fake-key'
 
 # Enables DCDO settings via cookies
 use_cookie_dcdo: true
+
+# aiproxy credentials
+aiproxy_api_key: !Secret
```

Comment:
please set this to be blank, just for the test environment. this ensures we never accidentally access aiproxy from our UI tests, which would be costly.